### ISSUE TITLE
Temperature reads 25 C or ambient then -127 C at random intervals on two of three ispindels built.

### ISSUE THREAD

**Issue by yakam-doolday** [universam1/iSpindel#182]
Hi,
Its not the temp probe or the rest of the circuit, I have narrowed it down the wemos, but i don't know whats wrong, anybody got any ideas?

I have used the pcb thats available and tested all three boards with the wemos that definitely works.
The two that work intermittently read the correct angle all the time but the temp suddenly goes from ambient say 25 degrees to -127 degree randomly, and the wemos won't save the wifi configuration either.
Please help, I am not an expert in electronics or computing but have managed to make one work.

**Comment by yakam-doolday** [comment]
Thanks for your reply ollinator2000.
The 18b20 sensor the resistors and the diode are all on a separate board that the wemos/GY-521 plug into.  All 3 of these boards work. However 2 of the wemos/GY-521 do not wok reliably.  I will try to capture the data from the serial monitor and post here this weekend.

**Comment by yakam-doolday** [comment]
Whoops pressed the wrong button.

**Comment by yakam-doolday** [comment]
After 10hrs Investigating this, I've found and fixed the problem.
The batch of 5 wemos'es I ordered from china, all had the ESP8266 sub board soldered to the wemos board with the absolute minimum of solder if any. The A0 pin was just resting on the pad hence the intermittent temperature readings.  Got my magnifying glass out and resoldered all 16 connections to wemos board.  Now working.
Everyone beware cheap isn't always better. Cheap just wasted about 10 hours of my life.
